src/components/button/__tests__/button.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { cleanup, fireEvent, render, screen } from '@testing-library/react';
+
+import { expect } from '@storybook/jest';
+
+import Button from '../index';
+
+import fn = jest.fn;
+
+describe('<Button>', () => {
```

Comment:
Does it makes sense to add test for the button variations (as24, ms24, disabled, sizes)?

Reply:
I'll add the test for disabled state. The theme, size are controlled by Chakra so I think it would not necessary for writing test cases